posts/2021-03-22-ldap-kerberos-21004beta.adoc

```diff
@@ -0,0 +1,252 @@
+
+<ldapRegistry id="LDAP" realm="SampleLdapADRealm" host="ldap_hostname" port="389" ignoreCase="true" baseDN="DC=example,DC=com" bindAuthMechanism="GSSAPI" krb5Principal="user1@EXAMPLE.COM" krb5TicketCache="${server.config.dir}/security/krb5-user1.cc" ldapType="Custom" />
+----
+
+The following example shows how to configure an LDAP Registry using a keytab and custom Kerberos config file:
+
+[source, xml]
+----
+<kerberos keytab= configFile="${server.config.dir}/security/krb5.conf"/>
+
+<ldapRegistry id="LDAP" realm="SampleLdapADRealm" host="ldap_hostname" port="389" ignoreCase="true" baseDN="DC=example,DC=com" bindAuthMechanism="GSSAPI" krb5Principal="user1@EXAMPLE.COM" ldapType="Custom" />
+----
+
+For more information on `LdapRegistry`, see the link:https://openliberty.io/docs/latest/reference/feature/ldapRegistry-3.0.html[LDAP User Registry documentation].
+
+To enable this new beta function in your app, pull the All Beta Features package and add the LDAP Registry 3.0 feature to your `server.xml` file:
+
+[source, xml]
+----
+<featureManager>
+  <feature>ldapRegistry-3.0</feature>
+</featureManager>
+----
+
+=== Try it now
+
+To try out these features, just update your build tools to pull the Open Liberty All Beta Features package instead of the main release. The beta works with Java SE 15, Java SE 11, or Java SE 8.
+
+If you're using link:{url-prefix}/guides/maven-intro.html[Maven]:
+
+1. Add the following dependency to your `pom.xml` file:
++
+[source,xml]
+----
+<dependency>
+  <groupId>io.openliberty.beta</groupId>
+  <artifactId>openliberty-runtime</artifactId>
+  <version>21.0.0.4-beta</version>
+  <type>pom</type>
+</dependency>
+----
++
+2. Add a dependency for the beta version of each of the APIs that you'd like to try. For example, to try MicroProfile Config 2.0:
++
+[source,xml]
+----
+<dependency>
+  <groupId>org.eclipse.microprofile.config</groupId>
+  <artifactId>microprofile-config-api</artifactId>
+  <version>2.0</version>
+  <scope>provided</scope>
+</dependency>
+----
++
+3. Add add the following `runtimeArtifact` section to the `configuration` section of your `pom.xml` file:
++
+[source,xml]
+----
+<runtimeArtifact>
+  <groupId>io.openliberty.beta</groupId>
+  <artifactId>openliberty-runtime</artifactId>
+  <version>21.0.0.4-beta</version>
+  <type>zip</type>
+</runtimeArtifact>
+----
+
+Or for link:{url-prefix}/guides/gradle-intro.html[Gradle]:
+
+[source,gradle]
+----
+dependencies {
+    libertyRuntime group: 'io.openliberty.beta', name: 'openliberty-runtime', version: '[21.0.0.4-beta,)'
+}
+----
+
+Or take a look at our link:{url-prefix}/downloads/#runtime_betas[Downloads page].
+
+[#jakarta]
+== Jakarta EE 9 Beta Features package
+
+As of the 21.0.0.2-beta release, Open Liberty is the first vendor product to be Jakarta EE Web Profile 9.0 compatible. With the recent 21.0.0.3-beta release, Open Liberty is the first vendor product to be added to the Jakarta EE Platform 9.0 compatibility list.
```

Comment:
Make "Jakarta EE Platform 9.0 compatibility list" a link to https://jakarta.ee/compatibility/#tab-9

Reply:
Done